core/src/main/java/apoc/ApocSettings.java

```diff
@@ -33,7 +33,7 @@
     public static final Setting<Boolean> apoc_ttl_enabled = newBuilder(APOC_TTL_ENABLED, BOOL, false ).build();
 
     @Description("maximum number of nodes to be deleted during one iteration")
-    public static final Setting<Long> apoc_ttl_limit = newBuilder(APOC_TTL_LIMIT, LONG, 1000l ).build();
+    public static final Setting<Long> apoc_ttl_limit = newBuilder(APOC_TTL_LIMIT, LONG, 0l ).build();
```

Comment:
lascia 1000 ma cambialo così
```suggestion
    public static final Setting<Long> apoc_ttl_limit = newBuilder(APOC_TTL_LIMIT, LONG, 0L ).build();
```